app/controllers/settings/contact-info.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  actions: {
+    updateContactInfo() {
+      this.set('isLoading', true);
+      let currentUser = this.get('model');
+      currentUser.save()
+        .then(() => {
+          this.set('isLoading', false);
+          this.get('notify', 'Updated information successfully');
```

Comment:
@sumedh123 I don't think this is the right way to display the success notification, also it's not working either. No notification appears on deployment link after updation.